### ISSUE TITLE
Tail dependence coefficients

### ISSUE THREAD

**Issue by tnagler** [vinecopulib/vinecopulib#578]
We should implement `Bicop::taildep_coefs()` or something similar. If not implemented, it could return `NAN`. 

I'm not a huge fan of using/interpreting them in a data analysis, but they have a somewhat important didactical value.

**Comment by tvatter** [comment]
Yup, it's something that was also requested by a pyvinecopulib user [here](https://github.com/vinecopulib/pyvinecopulib/issues/121)

**Comment by tvatter** [comment]
Alright, I've started in https://github.com/vinecopulib/vinecopulib/tree/tail_dependence but boy this is annoying.

**Comment by tvatter** [comment]
https://github.com/vinecopulib/vinecopulib/pull/682 thank god for AI, closing this
